searchconsole/query.py

```diff
@@ -178,6 +178,26 @@ def search_type(self, search_type):
 
         return self
 
+    @utils.immutable
+    def dataState(self, dataState='final'):
+        """
+        Return a new query filtered by the specified dataState.
+        Args:
+            dataState (str): The dataState you would like to use for your report.
+                Possible values: 'final' (default - only finalized data), 'all' (finalized & fresh data).
```

Comment:
Could we give a bit more detail on what "data state" is? Just a sentence or two on fresh compared to final data would really help. It would be great if you would wrap the lines at 80 characters too.